qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -231,11 +254,68 @@ def __init__(self,
             get_cmd=":FUNCtion:IMPedance:RANGe?",
             set_cmd=":FUNCtion:IMPedance:RANGe {}",
             unit='Ohm',
-            vals=Enum(0.1, 1, 10, 100, 300, 1000, 3000, 10000, 30000, 100000),
+            vals=self._imp_range,
             docstring="Selects the impedance measurement range, also turns "
                       "the auto range function OFF."
```

Comment:
dang, i just noticed this! is this "auto range" related to the DC bias "autorange_enabled"? or is it related to impedance? (if it's the latter, then all is good, it's just that apparently the driver is missing a parameter that lets the user do "auto ranging" of the impeadance measurement; if it's the former, then we should change the set command of this parameter to call get() on the dc_bias_autorange_enabled parameter so that it gets updated when a new impedance range is being set)

Reply:
This parameter sets the range for impedance measurement. There is another command which sets auto range for impedance - `:FUNCtion:IMPedance:RANGe:AUTO {ON|OFF|1|0}` which is not implemented in this driver.